### ISSUE TITLE
Existing links or links pasted into remnote are turning vertically

### ISSUE THREAD

**Issue by sanghosuh** [remnoteio/remnote-issues#299]
**Describe the bug**
(Not all but some) Links are shown vertically. Never saw this before. 

**To Reproduce**
1. Go to any remonote
2. Copy some URL (some URLs are okay; maybe it occurs when the title of an HTML isn't accessible or has some issue)
3. Paste into remnote

**Expected behavior**
Pasted links may turn vertically

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.7
 - Browser Chrome
 - Version 86.0.4240.193 (Official Build) (x86_64)

**Comment by mfranzs** [comment]
Thanks for the report. Can you please share an example link for which this happens?

**Comment by sanghosuh** [comment]
Hey, sorry for the late reply. Here is one link: 
https://petsymposium.org/2020/files/papers/issue1/popets-2020-0006.pdf 

I am sure you'll figure it out but to me, it doesn't seem like it has to do with links. Also, for some reason, on my Mac Mini, links are shown correctly. It's when I view it on my Macbook Pro that (some) links are shown vertically. I've updated my Chrome browser on Mac Mini to see if it has to do with the browser but the links are still shown vertically. (I did this because I was wondering if you did something to solve this bug on your recent December update but if the fix worked only for the Chrome version I mentioned above. I had updated Chrome on my Macbook Pro but not on Mac Mini). Thanks a lot for working on this, Martin!
